<launch>
    <arg name="env" default="uav_follow_trajectory_task_env"/>
    <arg name="agent" default="ddpg"/>

    <rosparam
        command="load"
        file="$(find ros_gym)/cfg/$(arg env).yaml" />

    <node
        pkg="ros_gym"
        name="ros_gym_node"
        type="ros_gym_node.py"
        output="screen">
    </node>

    <include file="$(find rl_agents)/launch/$(arg agent).launch">
    </include>
</launch>
